stories/Paginate.stories.tsx

```diff
@@ -24,14 +24,18 @@ const Template: Story<Props> = (args) => {
 
   const handlePagination = (value: number) => setCurrent(value);
 
+  const pages = usePagination({
+    range: args.range,
+    total: args.total,
+    current: current,
+    components: args.components,
+    handlePagination: handlePagination
+  });
+
   return (
-    <Paginate
-      current={current}
-      handlePagination={handlePagination}
-      range={args.range}
-      total={args.total}
-      components={args.components}
-    />
+    <>
+      { pages }
+    </>
```

Comment:
Let's better add another template rather override the default one, so developers can check the docs whether to use this hook or the Paginate component

Reply:
On it 👍.

Reply:
On it 👍.